src/posts/2023-09-12-cilium-hubble/index.md

```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-09-12T11:36:00.000Z'
+title: 'Tutorial: How to Use Cilium Hubble for Observability in CNI Chaining Mode (Part 1)'
+ogImage: ogimage.webp
+ogSummary: 'Learn about Cilium Hubble and how to utilize it for network observability without deploying Cilium, by leveraging CNI chaining'
+categories:
+  - How-To
+externalUrl: 'https://isovalent.com/blog/post/cilium-hubble-observability-cni-chaining-mode/'
+tags:
+  - Community
```

Comment:
```suggestion
  - How-To
```

Reply:
Hmm, i think it should be tagged community since it's external and there's already a category for How-To, that's why I left it that way. Unless we want to make categories and tags the same thing? What do you think?

Reply:
Sorry thought this was the category not the tag